SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Eiffel-community vulnerability and Security reporting
```

Comment:
Should we also include response in the heading since the doc talks about how the responses are done?

Reply:
Fixed